paddlets/datasets/tsdataset.py

```diff
@@ -271,7 +271,7 @@ def to_dataframe(self, copy: bool=True) -> pd.DataFrame:
         else:
             return self.data
     
-    def to_numpy(self, copy: bool=True) -> np.ndarray:
+    def to_numpy(self, copy: Optional[bool]=True) -> np.ndarray:
```

Comment:
Line274 copy should Not be Optional, refers to https://docs.python.org/3/library/typing.html#typing.Optional